openedx/features/marketplace/models.py

```diff
@@ -1,6 +1,33 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 
+from django.contrib.auth.models import User
 from django.db import models
 
-# Create your models here.
+from openedx.custom.db.fields import MultiSelectWithOtherField
+from openedx.features.idea.models import VisualAttachment, Location, OrganizationBase
+from openedx.features.marketplace.constants import (ORGANIZATION_SECTOR_CHOICES, ORGANIZATIONAL_PROBLEM_CHOICES,
+                                                    USER_SERVICES)
+
+
+class Challenge(OrganizationBase, Location, VisualAttachment):
+    user = models.ForeignKey(User, related_name='challenges', related_query_name='challenge', on_delete=models.CASCADE)
+    organization_sector = MultiSelectWithOtherField(other_max_length=50, choices=ORGANIZATION_SECTOR_CHOICES,
+                                                    verbose_name='Which sector is your organization working in?')
+
+    organizational_problems = MultiSelectWithOtherField(other_max_length=50, choices=ORGANIZATIONAL_PROBLEM_CHOICES,
+                                                        verbose_name='Current Organizational Problems')
+
+    description = models.TextField(blank=False, verbose_name='Brief Description of Challenges')
+
+    approch_to_address = models.TextField(blank=False,
+                                          verbose_name='How has your organization already tried to address these challenges?')
+
+    resources_currently_using = models.TextField(blank=False,
+                                                 verbose_name='What tools or resources are you currently using?')
+
+    user_services = MultiSelectWithOtherField(other_max_length=50, choices=USER_SERVICES,
```

Comment:
organization_services is more relevant here.

Reply:
@ahmed-faraz46 these are the services which a user can offer, so it is more relevant to that.

Reply:
My bad I misinterpreted it.